projects/reservoir/fv3/era_process.py

```diff
@@ -37,18 +62,34 @@ def main(args):
     setup_fregrid()
```

Comment:
I missed this in the previous PR, but the download locations for the data saved in this function are also hard coded. I'd change this to save them in a temporary directory (see other comment about this).